vendor/k8s.io/kubernetes/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -219,7 +223,10 @@ func ClusterRoles() []rbac.ClusterRole {
 
 				rbac.NewRule(ReadWrite...).Groups(extensionsGroup).Resources("daemonsets",
 					"deployments", "deployments/scale", "deployments/rollback", "ingresses",
-					"replicasets", "replicasets/scale", "replicationcontrollers/scale").RuleOrDie(),
+					"replicasets", "replicasets/scale", "replicationcontrollers/scale",
+					"networkpolicies").RuleOrDie(),
+
+				rbac.NewRule(ReadWrite...).Groups(networkingGroup).Resources("networkpolicies").RuleOrDie(),
```

Comment:
Seems a bit powerful for edit?

Reply:
The description says that "admin" is just "edit plus the power to grant permissions to other users", and the two have identical rules other than the two under "// additional admin powers" in "admin"

Reply:
this is in vendor/ shouldn't it go via a backport from upstream PR anyway ?